packages/loader/container-loader/README.md

```diff
@@ -178,3 +181,11 @@ Hosts can also force readonly-mode for a container via calling `Container.forceR
 
 ### `storageOnly`
 Storage-only mode is a readonly mode in which the container does not connect to the delta stream and is unable to submit or recieve ops. This is useful for viewing a specific version of a document.
+
+## Dirty events
+Container runtime (IRuntime) communicates with Container about state of the container. In response, Container will raise two events - `"dirty"` and `"saved"` events. Transitions between twese two events signify presence (or lack of) user changes that were not saved to storage. In other words, if container is dirty, closing it at that moment will result in data loss from user perspective, as not all user changes made it to storage.
+This information can be used by host to build appropriate UX that allows user to be confident in the platform. For example, host may chose to show "Your changes are not saved. Are you sure you want to close document/tab/...?" kind of dialong on closing. instead (or in addition to) host may chose to show "Saving..." and "Saved" text somewhere in UX. Coupled with lack of connectvity to ordering servive (and appropriate notificaiton to the user) that may creeate enough continuos notificaiton to user not to require blocking dialog on closing.
```

Comment:
```suggestion
This information can be used by a host to build appropriate UX that allows user to be confident in the platform. For example, a host may chose to show a dialog asking the user if they want to save their changes before closing. Instead of, or in addition to this, the host may choose to show "Saving..." and "Saved" text somewhere in UX. Coupled with lack of connectivity to ordering service (and appropriate notification to the user) that may create enough continuous notification to user not to require a blocking dialog on closing.
```